src/utilities/get-issue-numbers.ts

```diff
@@ -17,6 +17,7 @@ export function getIssueNumbers(text: string): number[] {
       issueNumbers.push(issueNumberOrNaN);
     }
   });
+  const issueNumbersWithoutDuplicate = [...new Set(issueNumbers)];
```

Comment:
This can be made cleaner, rather than keeping the original array, then converting to a set then converting back to an array. Remove the first array (issueNumbers) and push into the set, then finally return the values of the Set as an array.